spiralfilm/core.py

```
@@ -505,36 +507,22 @@ def num_tokens(self, messages: list[dict] | dict, model=None):
-            # "gpt-4-0314",
-            "gpt-4-32k-0314",
-            "gpt-4-0613",
-            "gpt-4-32k-0613",
-        }:
+        
+        if model.startswith("gpt-4o"):
+            tokens_per_message = 3
+            tokens_per_name = 1
+        elif model.startswith("gpt-4"):
             tokens_per_message = 3
             tokens_per_name = 1
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)